Sebastian-Martinez/tarea7/portada-react/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import PrincipalTitle from '../PrincipalTitle/PrincipalTitle'
+import TopTitle from '../TopTitle/TopTitle'
+import './Header.css'
+
+
+const Header = () => {
+  return (
+    <div className='header'>
```

Comment:
@jhonmartinezG you should use header tag

Reply:
fixed in the new commit